### PR TITLE
Project Page with minimum styling. Routing on navbar and dashboard fix

### PR DESCRIPTION
**Pull request by MalcolmAG**
Created a funding/startup page with minimum styling. 
Also made changes to navbar to introduce routing.
Additional small fix on UserDashboard

### CODE REVIEW

**Review by bonnieli — COMMENTED** [review]
lgtm!

File: client/src/pages/Project.js [hunk 1]
```diff
@@ -0,0 +1,229 @@
...
+      <Typography variant="h2">
+        <Box fontWeight="fontWeightMedium">{proj.title}</Box>
+      </Typography>
+      <Typography color="textSecondary">{proj.subtitle}</Typography>
+    </div>
+  );
+}
+
+function ProjectPanel(props) {
+  const [value, setValue] = React.useState(0);
```
Comment (tmclean15) [inline client/src/pages/Project.js:107]: It's a bit cleaner to import useState at the top, i.e. `import React, { useState} from "react"`

**Review by tmclean15 — APPROVED** [review]
Looks good!

File: client/src/pages/Project.js (hunk 1 above)
Comment (MalcolmAG) [reply client/src/pages/Project.js:107]: Thanks, change is made

**Pull request merged**